src/main/java/uk/gov/hmcts/darts/common/repository/TranscriptionRepository.java

```diff
@@ -75,4 +77,74 @@ List<TranscriptionEntity> findByHearingIdTypeStartAndEndAndIsManualAndNotStatus(
     );
 
     List<TranscriptionEntity> findByIdIn(List<Integer> transcriptionIds);
+
+    @Query("""
+            SELECT new uk.gov.hmcts.darts.transcriptions.model.TranscriptionSearchResult(
+                t.id,
+                cc.caseNumber,
+                cth.id,
+                h.hearingDate,
+                t.createdDateTime,
+                ts.id,
+                t.isManualTranscription)
+            FROM TranscriptionEntity t
+            JOIN t.hearings h
+            JOIN h.courtCase cc
+            JOIN h.courtroom cr
+            JOIN cr.courthouse cth
+            JOIN t.transcriptionStatus ts
+            JOIN t.createdBy ua
```

Comment:
how worried are we regarding performance here?

e.g. if the requestedBy is not provided in the search, then there would be no point in joining the UserAccount table in the query. Perhaps the performance hit is negligible?

Reply:
The performance seems quite good against 5000 records in my local postgres  about 1 second.  So in the absence any performance requirements I think that seems ok.